Tests/LiveViewNativeStylesheetMacrosTests/MacroTests.swift

```diff
@@ -27,12 +27,97 @@ final class MacroTests: XCTestCase {
+                        try "[" .utf8.parse(&input)
+                        let copy = input
+                        #if os(iOS) || os(macOS)
+                        do {
+                           return try
+                        _ThrowingParse({ (wildcard: Double?, labelled: Double?) -> Output in
+
+                            return Output.init(
+                                wildcard, height: labelled
+                            )
+                            }) {
+                            // Parse the wildcard arguments
+                            Parse({ width -> Double? in
+                            (width)
+                                }) {
+                            Whitespace()
+                            Double.parser(in: context)
+                            Whitespace()
+
+                            Whitespace()
+                            }
+
+                            // Parse the labelled arguments
+                            OneOf {
+                            Parse {
+                                Whitespace()
+                                "," .utf8
+                                Whitespace()
+                                "[" .utf8
+                                Many(
+                                    into: (Double?.none)
+                                ) { (result: inout Double?, argument: Double?) in
+                                    result = (
+                                        result ?? argument
+                                    )
+                                } element: {
+                                    Whitespace()
+                                    OneOf {
+                                        Parse({ value -> Double? in
+                            (value)
+                                            }) {
+                            "height:" .utf8
+                            Whitespace()
+                            Double.parser(in: context)
+                                        }
+                                        _ErrorParse {
+                                            Identifier().map(ArgumentParseError.unknownArgument)
+                                            ":" .utf8
+                                            Whitespace()
+                                        }
+                                    }
+                                    Whitespace()
+                                } separator: {
+                                    "," .utf8
+                                } terminator: {
+                                    "]" .utf8
+                                }
+                            }
+                            Always((Double?.none))
+                            }
+                            "]" .utf8
+                        }
+                           .parse(&input)} catch {
+                           input = copy}
+            #endif
```

Comment:
Nit: There's some weird indentation in some of this. Unclear if that's desired.

Reply:
The macro messes up the formatting a bit. I can try to clean up the formatting in the generated code, but I think that'd belong in a separate PR.